packages/react-native-editor/ios/Podfile.lock

```diff
@@ -416,9 +416,9 @@ SPEC CHECKSUMS:
   react-native-get-random-values: 03edb8dcc2d3f43e55aa67ea13b61b6723bbf047
   react-native-keyboard-aware-scroll-view: 6cb84879bf07e4cc1caed18b11fb928e142adac6
   react-native-safe-area: c9cf765aa2dd96159476a99633e7d462ce5bb94f
-  react-native-safe-area-context: c8da723dcddabe15afe95c9d31c56a0cf2b0141c
+  react-native-safe-area-context: 133d77d86b13cf694e014ab03dc2944ee3125d97
```

Comment:
@hypest, I had this file modified locally. Is it fine to apply those changes?

Reply:
Hmm, it's possibly OK as there might be a trivial change that caused the checksum change, but I'm far from an iOS expert so, let me ping @ceyhun for a look. Ceyhun, WHYT about this podfile.lock checksum change?

Reply:
I think these changes are not OK, nothing seems to relate with these changes in this PR (namely no `react-native-editor/package.json` changes). When I ran `npm run native preios` in this branch, these changes were reverted for me. I guess it has something to do with CocoaPods cache but I'm not really sure. Best would be to revert the changes to this file.

Reply:
Thanks, reverted in bac8b73.